package.json

```diff
@@ -75,6 +75,7 @@
     "jose": "^5.6.3",
     "js-cookie": "^3.0.5",
     "jszip": "^3.10.1",
+    "lucide-react": "^0.460.0",
```

Comment:
I've checked other icons on this repo and it's currently using 
https://unocss.dev/presets/icons

As a general practice, having 5 Shovels in your backpack is heavier that just 1 shovel.

Using the current tool, we have access to quite a choice of https://icones.js.org/collection/all?s=microphone